.ci/magic-modules/vars/validator_handwritten_files.txt

```diff
@@ -1,7 +1,9 @@
 resource_compute_instance.go
+resource_container_cluster.go
+resource_container_node_pool.go
 resource_google_project.go
 resource_sql_database_instance.go.erb
 resource_storage_bucket.go
 iam_folder.go
 iam_organization.go
-iam_project.go
\ No newline at end of file
+iam_project.go
```

Comment:
I don't understand why this line is showing up. The branch is sync'ed with master, and master version got this line. (https://github.com/GoogleCloudPlatform/magic-modules/blob/master/.ci/magic-modules/vars/validator_handwritten_files.txt)